app/controllers/test_controller.rb

```diff
@@ -16,7 +16,14 @@ def translations_answer
     @verb = set_verb
   end
 
-  def conjunctions
+  def conjugations
+  end
+
+  def conjugations_answer
+    @ik_correct = @verb.ovt_me == params[:ik_answer]
+    @wij_correct = @verb.ovt_you == params[:wij_answer]
+    @old_verb = @verb
```

Comment:
i think the
```
@old_verb = @ verb
@verb = set+verb
```

is something we also do in the other `answer` action, maybe we should make it a separate method, but we don't have to do it right now, just because we're also going to repeat it for the other `answer` action